InterventionSkillExample/main.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from fastapi import FastAPI
+
+app = FastAPI()
+
+model = None
+
+@app.post('/invoke')
+def run(params: dict):
+
+    try:
+        payload = params["payload"]
+        profile = payload["profiles"]
+        intervention = payload["intervention_id"]
+        message = ""
+        try:
+            message = "Successfully invoked action for given profile ID "+str(profile)+"for intervention "+str(intervention)
+        except Exception as e:
+            print("Error in invoking action")
+
+        print(datetime.datetime.now())
+        return {"response": {'message': message}}
+    except Exception as e:
+        print(e)
```

Comment:
Need context. Also,  we should throw a 400. If we are going to call another API or endpoint we should catch that as a 500x exception if we see server side errors return from the endpoint.